1/tom-stuart-captions.srt

```diff
@@ -5711,7 +5711,7 @@ to stay in the same order
 1230
 01:16:11,800 --> 01:16:13,740
 in the array, we
-can't use Rubys' sort by.
+can't use Ruby's sort_by.
```

Comment:
This looks good. @mudge had a different approach for method names, which was to put them in quotes (e.g. `Ruby's "sort by"`). It makes sense to differentiate keywords and method names somehow and both of these solutions work. I'll see what the other reviewers come back with before making a decision one way or the other.